docs/pages/desktop-access/active-directory-manual.mdx

```diff
@@ -682,6 +682,49 @@ Computer Configuration > Policies > Windows Settings > Security Settings > Local
 Double click the "Add workstations to domain" policy and ensure that the
 "Authenticated Users" group is not present.
 
+## Multiple Domains
+
+Each `windows_desktop_service` is designed to support connecting to hosts in a
+single Active Directory domain. If you have multiple independent domains, you
+can deploy multiple Teleport agents to service them.
+
+If you have multiple domains with a trust relationship between them, you can
+configure Teleport to perform PKI operations against a different domain.
+
+In order for this to work, the hosts that you want to connect to and the AD
+users that you want to connect as must reside in the same domain.
```

Comment:
> you can configure Teleport to perform PKI operations against a different domain.

> must reside in the same domain.

These two appear to contradict each other, consider whether there's a different and/or more precise way to word this.

Reply:
The users must be in the same domain as the computers. The PKI set up can be in a different domain. Open to suggestions for alternative ways to phrase it.

Reply:
```suggestion
Alternatively, if you have a pair of domains A and B with a trust relationship between them, you can
configure Teleport to perform PKI operations against domain A for users/hosts in domain B.

<Admonition type="warning" title="Supported Configuration">
  In order for this to work, the hosts that you want to connect to and the
  users that you want to connect as must reside in the same domain. Other
  configurations are not currently supported.
</Admonition>
```

Reply:
@ptgott what say you? I try to avoid fancy components and extra flair in favor of plain old text. Is this important enough to highlight?

Reply:
I think this particular passage works just fine without the Admonition. The second paragraph flows naturally from the first, so (at least theoretically) users should move from the first paragraph to the second without the need for a box.